### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Feb 25, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2945] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Peter update
2. Dong update
3. Jason update
4. Cold cases:
a. #1715 (Peter)
b. #1220 (Dong)
c. @balacij https://github.com/JacquesCarette/Drasil/issues/2853#issuecomment-1034427454 + https://github.com/JacquesCarette/Drasil/issues/2911#issuecomment-1049180439
5. Questions/Comments/Concerns/Discussion

**Comment by smiths** [comment]
@cd155 and @balacij, please add an issue to our agenda when you get a chance.

**Reply by balacij** [reply]
I've updated mine now. Thank you!

**Reply by cd155** [reply]
Added a new case.

**Comment by smiths** [comment]
We completed the agenda, except for only partially getting through 4.c.  We will start the next set of "cold issues" with 4.c.
